web/package.json

```diff
@@ -11,7 +11,10 @@
     "body-parser": "~1.17.1",
     "cookie-parser": "~1.4.3",
     "ejs": "~2.5.6",
-    "express": "~4.15.2"
+    "express": "~4.15.2",
+    "moment": "^2.18.1",
```

Comment:
If we are using moment for client-side code, please keep in mind that the library is huuuuuge. Maybe there are smaller alternatives.